website/docs/d/is_lb_listener.html.markdown

```diff
@@ -33,9 +33,11 @@ In addition to all argument references listed, you can access the following attr
 - `accept_proxy_protocol` - (Boolean) If set to `true`, this listener will accept and forward PROXY protocol information. Supported by load balancers in the `application` family (otherwise always `false`). Additional restrictions:- If this listener has `https_redirect` specified, its `accept_proxy_protocol` value must  match the `accept_proxy_protocol` value of the `https_redirect` listener.- If this listener is the target of another listener's `https_redirect`, its  `accept_proxy_protocol` value must match that listener's `accept_proxy_protocol` value.
 
 - `certificate_instance` - (List) The certificate instance used for SSL termination. It is applicable only to `https`protocol.
-Nested scheme for `certificate_instance`:
+	Nested scheme for `certificate_instance`:
 	- `crn` - (String) The CRN for this certificate instance.
 
+	-> **NOTE:** Migrate your load balancer certificates from Certificate Manager to Secrets Manager as the certificate CRN from Certificate Manager is deprecated.
```

Comment:
Certificate Manager is deprecated. Migrate your load balancer certificates from Certificate Manager to Secrets Manager.